### DISCUSSION TITLE
colours of op labels disappeared after todays update

### DISCUSSION THREAD

**Discussion by MarcVdH** [cables-gl/cables_docs#600] [! Welcome - readme first!]
thanks for making Cables even better !   I noticed two minor issues after todays update....
![image](https://github.com/cables-gl/cables_docs/assets/126583906/ff368df1-d9ff-404a-8ed5-e64413b0777b)

**Comment by pandrr** [comment]
thanks ! fixed on dev , will move to live soon!
